docs/1-general-configuration.md

```diff
@@ -42,12 +42,22 @@ config.site_title_image = ->(context) { context.current_user.company.logo_url }
 
 ## Internationalization (I18n)
 
+### Localize Format For Dates and Times
+
+Active Admin uses `:long` as default localize format for dates and times.
+If you want, you can customize it.
+
+```ruby
+config.localize_format = :short
+```
+
```

Comment:
If you add a headline above, you need to add one underneath for the next section